tools/docker-compose-datasets-server.yml

```diff
@@ -28,6 +28,10 @@ services:
     build:
       context: ..
       dockerfile: services/admin/Dockerfile
+    volumes:
+      - assets:${ASSETS_STORAGE_DIRECTORY-/assets}:rw
+      - duckdb-index:${DUCKDB_INDEX_CACHE_DIRECTORY-/duckdb-index}:rw
```

Comment:
Do you know if the volume for statistics is missing here? Same for changes on line 52-55